docs/sources/setup-grafana/installation/_index.md

```diff
@@ -5,6 +5,8 @@ aliases:
   - ../installation/installation/
   - ../installation/requirements/
   - /docs/grafana/v2.1/installation/install/
+  - ../../installation/installation/rpm/
+  - ../../installation/rpm/
```

Comment:
I'm not sure what the intent is with the doubled `installation/` or why we have `../installation/` instead of `./`, seems this could just be the below (and the `../installation/` in the lines above could be replaced with `./`):
```suggestion
  - ./rpm/
```